File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    n = nums.length
    for(i =0 ; i<=n;i++){
        if(!nums.includes(i)){
            return i
        }
    }
    
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = max(nums)
        for i in range(n+1):
            if i not  in  nums:
                return i 
            elif n== 0 :
                return 1
            else :
                pass
        else :
            return n+1